server/src/main/java/de/uftos/services/TimetableService.java

```diff
@@ -168,10 +175,22 @@ public Timetable getById(String id) {
    */
   public Timetable create(TimetableRequestDto timetable) throws ResponseStatusException {
     if (timetable.name().isBlank()) {
-      throw new ResponseStatusException(HttpStatus.BAD_REQUEST, "The name of the timetable is blank.");
+      throw new ResponseStatusException(HttpStatus.BAD_REQUEST,
+          "The name of the timetable is blank.");
     }
     Consumer<TimetableSolutionDto> solverFinishedEvent = (solution) -> {
+      if (solution.hardScore() > 0) {
```

Comment:
This will never happen. The scores are alway <= 0 (0 -> all constraints could be satisfied).

Reply:
```suggestion
      if (solution.hardScore() < 0) {
```